api/app/src/routes/medical/models/organization.ts

```diff
@@ -0,0 +1,23 @@
+import { z } from "zod";
+import { addressSchema } from "./address";
+
+export enum OrgType {
+  acuteCare = "acuteCare",
+  ambulatory = "ambulatory",
+  hospital = "hospital",
+  labSystems = "labSystems",
+  pharmacy = "pharmacy",
+  postAcuteCare = "postAcuteCare",
+  testing = "testing",
```

Comment:
no idea what "testing" is for in CW - but we probably shouldn't expose this to cxs